test/e2e/storage/vsphere_volume_placement.go

```diff
@@ -77,11 +77,13 @@ var _ = SIGDescribe("Volume Placement", func() {
 		2. Delete VMDK volume
 	*/
 	framework.AddCleanupAction(func() {
-		if len(node1KeyValueLabel) > 0 {
-			framework.RemoveLabelOffNode(c, node1Name, "vsphere_e2e_label")
-		}
-		if len(node2KeyValueLabel) > 0 {
-			framework.RemoveLabelOffNode(c, node2Name, "vsphere_e2e_label")
+		if c == nil {
```

Comment:
This should be `c != nil`.

Suggest changing this one to `len(ns)` and add the comments too

Reply:
I'm sorry for missing this, have updated.